### DISCUSSION TITLE
[PLUGIN SUGGESTION] add select2 search

### DISCUSSION THREAD

**Discussion by tamilventhan** [TiagoSilvaPereira/vemto-issues#139] [Ideas & Suggestions]
select2 search option will very usefull and will be fulfilled max requirement before complting developement cycle

**Comment by TiagoSilvaPereira** [comment]
Hi @tamilventhan thank you for your suggestion... as select2 is a very opinionated library (jquery, own CSS, etc), I think it best fits as a Vemto plugin. I want to bring Vemto plugin capabilities in the next months
